### PR TITLE
Updating Functions

### PR DESCRIPTION
**Pull request by Neodevils**
Updating functions from Y→N.

### CODE REVIEW

**Comment by Neodevils** [comment]
The app had issues with merging, it has been fixed on `a42bccb`.

File: src/functions/usdvolume.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+description: Changes volume for current playing song.
 ---
 
 # $volume
 
-This function can change the volume of the current playing song. The minimum number is 0 and the highest number is what ever you want \(Just be careful as it can cause earrape\)
+This function can change the volume of the current playing song. The minimum number is 0 and the highest number is what ever you want~
 
-Function Usage: 
+> P.S. : I hope you're not thinking about earrape...
```
Comment (Leref) [inline src/functions/usdvolume.md]: This isn't needed.

**Review by Leref — CHANGES_REQUESTED** [review]

File: src/functions/usdvanityuses.md [hunk 2]
```diff
@@ -1,11 +1,30 @@
...
+
 # $vanityUses
 
-This function returns how many users joined through the vanity link
+This function returns how many users joined through the vanity link[^1]
+
+### Usage
+
+```php
+$vanityUses[guild id?]
```
Comment (Leref) [inline src/functions/usdvanityuses.md]: guildID proper.

File: src/functions/usdvoiceid.md [hunk 3]
```diff
@@ -1,38 +1,33 @@
...
 
 This function simply returns the voice channel ID that the user is in
 
-Heres the usage:
+### Usage
 
-```text
-$voiceID[userID (optional)]
+```php
+$voiceID[user id?]
```
Comment (Leref) [inline src/functions/usdvoiceid.md]: userID proper.

**Review by Leref — CHANGES_REQUESTED** [review]

**Review by Leref — APPROVED** [review]

**Pull request merged**